### PR TITLE
Fix EffActionBar NullPointException when message is null

### PR DESCRIPTION
**Pull request by Fusezion**
### Description

This PR aims to fix the NullPointException caused when message is null in EffActionBar
in addition I've made `to %players%` optional as players have been asking for this for awhile now and is a minor change

---
**Target Minecraft Versions:**  any
**Requirements:**  any
**Related Issues:**  #5776

### CODE REVIEW

File: src/main/java/ch/njol/skript/effects/EffActionBar.java [hunk 1]
```diff
@@ -54,25 +54,28 @@ public class EffActionBar extends Effect {
...
 	@SuppressWarnings("deprecation")
 	@Override
-	protected void execute(final Event e) {
-		String msg = message.getSingle(e);
+	protected void execute(Event event) {
+		String msg = message.getSingle(event);
 		assert msg != null;
+		if (msg == null) {
+			return;
+		}
```
Comment (AyhamAl-Ali) [inline src/main/java/ch/njol/skript/effects/EffActionBar.java:70]:
```suggestion
		if (msg == null)
			return;
```

**Review by AyhamAl-Ali — APPROVED** [review]

File: src/main/java/ch/njol/skript/effects/EffActionBar.java [hunk 2]
```diff
@@ -54,25 +54,27 @@ public class EffActionBar extends Effect {
...
 		return true;
 	}
 
 	@SuppressWarnings("deprecation")
 	@Override
-	protected void execute(final Event e) {
-		String msg = message.getSingle(e);
+	protected void execute(Event event) {
+		String msg = message.getSingle(event);
 		assert msg != null;
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/effects/EffActionBar.java:67]:
```suggestion
```
bit of a silly assertion

**Review by APickledWalrus — APPROVED** [review]

File: src/main/java/ch/njol/skript/effects/EffActionBar.java (hunk 2 above)
Comment (Fusezion) [reply src/main/java/ch/njol/skript/effects/EffActionBar.java:67]: Wasn't sure if we wanted to remove it or not

File: src/main/java/ch/njol/skript/effects/EffActionBar.java [hunk 3]
```diff
@@ -52,27 +52,28 @@ public class EffActionBar extends Effect {
 	@SuppressWarnings("null")
 	private Expression<Player> recipients;
 
-	@SuppressWarnings({"unchecked", "null"})
 	@Override
-	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final SkriptParser.ParseResult parser) {
+	@SuppressWarnings({"unchecked", "null"})
```
Comment (TheLimeGlass) [inline src/main/java/ch/njol/skript/effects/EffActionBar.java:56]:
```suggestion
	@SuppressWarnings("unchecked")
```

**Review by TheLimeGlass — CHANGES_REQUESTED** [review]

File: src/main/java/ch/njol/skript/effects/EffActionBar.java [hunk 4]
```diff
@@ -43,7 +43,7 @@
 public class EffActionBar extends Effect {
 
 	static {
-		Skript.registerEffect(EffActionBar.class, "send [the] action[ ]bar [with text] %string% to %players%");
+		Skript.registerEffect(EffActionBar.class, "send [the] action[ ]bar [with text] %string% [to %players%]");
 	}
 
 	@SuppressWarnings("null")
```
Comment (TheLimeGlass) [inline src/main/java/ch/njol/skript/effects/EffActionBar.java:49]:
```suggestion
```

File: src/main/java/ch/njol/skript/effects/EffActionBar.java [hunk 5]
```diff
@@ -52,27 +52,28 @@ public class EffActionBar extends Effect {
...
```
Comment (TheLimeGlass) [inline src/main/java/ch/njol/skript/effects/EffActionBar.java:52]:
```suggestion
```

File: src/main/java/ch/njol/skript/effects/EffActionBar.java [hunk 6]
```diff
@@ -52,27 +52,28 @@ public class EffActionBar extends Effect {
...
 	private Expression<Player> recipients;
 
-	@SuppressWarnings({"unchecked", "null"})
 	@Override
-	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final SkriptParser.ParseResult parser) {
+	@SuppressWarnings("unchecked")
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parser) {
```
Comment (TheLimeGlass) [inline src/main/java/ch/njol/skript/effects/EffActionBar.java:57]:
```suggestion
	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
```

**Review by TheLimeGlass — CHANGES_REQUESTED** [review]

**Review by TheLimeGlass — APPROVED** [review]

**Pull request merged**